proteuslib/edb/tests/test_db_api.py

```diff
@@ -72,3 +78,90 @@ def test_edb_load_convention():
     # we rely on it in the ElectrolyteDB.load() method
     for wrapper_class in (Component, Reaction, Base):
         assert wrapper_class.__name__.lower() in ElectrolyteDB._known_collections
+
+
+def insert_reactions(collection, data):
+    for obj in data:
+        collection.insert_one(obj)
+
+
+# Data for get_reactions tests
+
+data1 = [{
+    "stoichiometry": {"Liq": {"H2O": -1, "CO2": -1, "H2CO3": 1}},
+    "heat_of_reaction": "constant_dh_rxn",
+    "equilibrium_constant": "van_t_hoff",
+    "equilibrium_form": "log_power_law",
+    "concentration_form": "ConcentrationForm.molarity",
+    "parameter_data": {
+        "dh_rxn_ref": [{"v": 0, "u": "kJ/mol", "i": 0}],
+        "k_eq_ref": [{"v": 0.0017, "u": "m**3/mol", "i": 0}],
+        "T_eq_ref": [{"v": 300, "u": "K", "i": 0}],
+    },
+    "type": "equilibrium",
+    "name": "CO2_to_H2CO3",
+    "components": ["CO2", "H2CO3"],
+    "reactant_elements": ["C", "O", "H"],
+}, {
+    "stoichiometry": {"Liq": {"H2O": -1, "H +": 1, "OH -": 1}},
+    "heat_of_reaction": "constant_dh_rxn",
+    "equilibrium_constant": "van_t_hoff_aqueous",
+    "equilibrium_form": "log_power_law",
+    "concentration_form": "ConcentrationForm.molarity",
+    "parameter_data": {
+        "dh_rxn_ref": [{"v": 55.83, "u": "kJ/mol", "i": 0}],
+        "ds_rxn_ref": [{"v": -80.7, "u": "J/mol/K", "i": 0}],
+    },
+    "type": "equilibrium",
+    "name": "H2O_Kw",
+    "components": ["H2O", "Kw"],
```

Comment:
I am a little confused by what the "components" is in this structure, but I don't think it matters for the time being.

Reply:
that's a bug. components is generated by splitting the name and Kw looked like the name of an element to the regular expression